app/models/host/managed.rb

```diff
@@ -295,6 +296,11 @@ def clear_data_on_build
     self.build_errors = nil
   end
 
+  def set_initiated_at
+    return unless build?
```

Comment:
Maybe I could replace with `return unless build? && self.initiated_at.nil?`

Reply:
Nice find, mind to call this method from `setBuild` actually so the code (one line) is re-used actually?

Reply:
Not sure about `self.initiated_at.nil?` - it can be already set from previous provisioning, we need to be able to overwrite. Or what is the idea behind this?

Reply:
Yeah, I don't think that's right either.  I started out without `&& self.initiated_at.nil?` but when the host finished building `initiated_at` was reset.  How should I set `self.initiated` when a new host is being built? (currently it only gets set when the 'Build' button is used).